person.rb

```diff
@@ -1,6 +1,13 @@
-class Person
-  def initialize(age, name = 'unknown', parent_permission: true)
-    @id = Random.rand(1..1000)
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class Person < Nameable
+  def initialize(age, name, parent_permission: true)
+    super
+    @id = Random.rand(1...1000)
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216038673-78b22969-f459-4104-b790-71a2c0a86219.png)

- [ ] When I try to create a person then I am facing this error which you can see in the image above :arrow_up: Kindly fix this error by adding parentheses after `super` such as `super()` so this error won't come :+1: